actors-library/src/main/java/com/truecaller/androidactors/SecureParameter.java

```diff
@@ -28,5 +28,5 @@
     int LEVEL_NULL_OR_EMPTY_STRING = 1;
     int LEVEL_FULL_INFO = 2;
 
-    int value() default LEVEL_FULL_INFO;
+    int value() default LEVEL_NULL_OR_EMPTY_STRING;
```

Comment:
If you change default value here, you should probably also change it in here: https://github.com/truecaller/android-actors-library/pull/15/commits/ad33c7189ae9158b92dc9236bc4788bcd6841ebe#diff-82a44d7932d2bb1c7309ac716c55765fR257